.github/workflows/tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Run tests
+on:
+    push:
+        branches-ignore:
+            - 'gh-pages'
```

Comment:
Will this work on push to master as well?

Reply:
I don't think so. There is a very similar thing in the workflow in master. We have the same `on.push` clause in that workflow and it does not run on pushes to maintenance branches.

Reply:
The push commit check runs on 5.0.x because it has a workflow file with `on.push`, but on 4.2.x there is no check on commit push since there is no workflow file in that branch. So I think every workflow file is local to its branch.